modules/sso-rbac/README.md

```diff
@@ -0,0 +1,93 @@
+This is a module which will configure `GKE RBAC` to sync with Google groups
+
+```
+module "gcp-sso-module" {
```

Comment:
please set module source from terraform registry, and also have the version line,

remove all following locals, data, provider blocks. just left one this module, put minimal role binding inside the module scopes,

remove cluster_location and cluster_name variable here and 2 examples you have as this module have no this variables anymore